server/src/test/java/org/opensearch/gateway/remote/RemoteClusterStateCleanupManagerTests.java

```diff
@@ -231,6 +269,7 @@ public void testDeleteClusterMetadata() throws IOException {
                     + ".dat"
             )
         );
+        verify(container).deleteBlobsIgnoringIfNotExists(List.of(new BlobPath().buildAsString() + index1Metadata.getUploadedFilename()));
```

Comment:
Considering the manifest 7 is is the latest manifest. Why the indicesRouting from manifest6, which has index1Metadata is deleted?

Reply:
Should we create our own variables for better understanding? Using the same variable make it difficult since the container and filename would be same.

Reply:
Created new path for remoteRouting table files.